src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -29,16 +32,19 @@ import org.spongycastle.crypto.AsymmetricCipherKeyPair
   * and `Status` exchange).
   * Once that's done it can send/receive messages with peer (HandshakedHandler.receive).
   */
+//FIXME: MessageHandler type should be configurable, this is dependant on PR 185
```

Comment:
Not necessarily, this could be fixed in a later PR, I left this comment to remind us of changing this for the last version of the `Network` module.